src/main/hugo/content/reference/yaml/_index.md

```diff
@@ -305,7 +309,29 @@ Otherwise, the [arbitrary class loading system](#arbitrary-class-loading-system)
 
 ### `content.in`
 
-**Type**: Traversable of Shape
+**Type**: Traversable of [shapeFilter](#shapefilter)
+
+### `property`
+
+**Type**: SpecMap
+
+**(Multi)Spec**
+
+| Mandatory keys | Optional keys      |
+|----------------|--------------------|
+| `type`         | `parameters`, `in` |
+
+### `property.type`
+
+Same as [type](#type)
+
+### `property.parameters`
+
+Same as [parameters](#parameters)
+
+### `property.in`
```

Comment:
This does not match what we discussed, I think.
I thought `in` was the **container** of `property`, not vice-versa